components/filter/events.tsx

```diff
@@ -199,24 +199,28 @@ const Event = ({ visible, event }: EventProps) => {
             />
 
             <div>
-              {event.CalendarType && (
-                <>
-                  <strong>Type:</strong> {event.CalendarType}{" "}
-                </>
-              )}
               {event.Presenter && (
-                <>
-                  <strong>Presenter: </strong>
-                  {event.Presenter}
-                </>
+                <EventDescItem
+                  label="Presenter"
+                  linkValue={event?.PresenterProfileUrl?.Url}
+                  value={event.Presenter}
+                />
               )}
               {event.City && CITY_MAP[event.City] && (
-                <span className="ml-3">
-                  <strong>Location: </strong>
-                  <CustomLink href={CITY_MAP[event.City]?.url}>
-                    {CITY_MAP[event.City]?.name}
-                  </CustomLink>
-                </span>
+                <EventDescItem
+                  label="Location"
+                  value={CITY_MAP[event.City]?.name}
+                  linkValue={CITY_MAP[event.City]?.url}
+                />
+              )}
+              {event.CalendarType && (
+                <EventDescItem label="Type" value={event.CalendarType} />
+              )}
+              {event.Category_f5a9cf4c_x002d_8228_x00 && (
```

Comment:
why we have this strange looking variable there?

Reply:
Not sure - this is what its called as it comes from SharePoint - I imagine its an internal field name.

Reply:
Is it possible to rename the variable name from Sharepoint?

Reply:
I doubt it - seems like something that was automatically added because there was already a category field.